File: java/src/main/java/com/dsi/laboreos/model/Estado.java
package com.dsi.laboreos.model;

public class Estado {
    private String nombre;
    private String descripcion;
    private Boolean estadoFinal;

    public Estado(String nombre, String descripcion, Boolean estadoFinal) {
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.estadoFinal = estadoFinal;
    }

    public String getNombre() {
        return nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public Boolean esFinal() {
        return estadoFinal;
    }
}

File: python/laboreos/models/Estado.py
from django.db import models


class Estado(models.Model):
    nombre = models.CharField(max_length=100, primary_key=True)
    descripcion = models.TextField()
    es_final = models.BooleanField(default=False)
    
    class Meta:
        db_table = 'estados'
    
    def __str__(self):
        return self.nombre

